common/authed.js

```diff
@@ -114,6 +130,7 @@ module.exports = {
     requireNoAuth,
     requireUserAuth,
     requireActiveUser,
+    requiredUnactivatedUser,
```

Comment:
Nitpicking but should this be called `requireUnactivatedUser`? (eg. no `d`)?

Reply:
Oh yes! Nitpick away